components/resume/JobDetailsModal.tsx

```
@@ -0,0 +1,160 @@
+'use client'
+
+import {
+	Dialog,
+	DialogContent,
+	DialogDescription,
+	DialogHeader,
+	DialogTitle,
```

ESLint

[error] Unexpected trailing comma.

(@stylistic/js/comma-dangle)